casacore/images/coordinates.py

```diff
@@ -292,7 +292,7 @@ def get_frame(self):
 
     def set_frame(self, val):
         # maybe uses measures here
-        #dm = measures();knonwframes = dm.listcodes(dm.frequency())["normal"]
+        #dm = measures();knonwframes = dm.list_codes(dm.frequency())["normal"]
```

Comment:
Just below is a hardcoded list: ["BARY", ...] . That list could be dynamic, from the referenced function. Except that gives a longer list:
```['REST', 'LSRK', 'LSRD', 'BARY', 'GEO', 'TOPO', 'GALACTO', 'LGROUP', 'CMB']```
. Not sure what @gervandiepen meant here, but let's leave it as a todo in the code.